docs/operator-guide/install-argocd.md

```diff
@@ -87,106 +87,107 @@ Follow the steps below to install Argo CD using Helm:
   </details>
 :::
 
-1. Check out the _values.yaml_ file sample of the Argo CD customization, which is based on the `HA mode without autoscaling`:
+1. Fork the [Add-Ons](https://github.com/epam/edp-cluster-add-ons/tree/main/argo-cd) repository to your personal account.
 
-    <details>
-    <summary><b>View: kubernetes-values.yaml</b></summary>
+2. Run the installation:
 
-      ```yaml
-      redis-ha:
-        enabled: true
+    ```bash
+    helm dependency update argo-cd
+    helm install argocd argo-cd -n argocd --create-namespace
+    ```
+
+3. Port-forward Argo CD service using kubectl:
+
+    ```bash
+    kubectl port-forward svc/argo-argocd-server 8080:80 -n argocd
+    ```
+
+4. Login to the Argo CD server in browser using login and password:
+
+    :::info
+      By default, to access the console with administrative privileges, use the following credentials:
+	    - URL: localhost:8080
+      -	Login: admin
+    :::
+
+    ```bash title="password"
+    kubectl get secret -n argocd argocd-initial-admin-secret --template={{.data.password}} | base64 -d
+    ```
+5. Integrate Argo CD with add-ons and install core add-ons:
```

Comment:
5. Integrate Argo CD with Add-Ons(add-ons-overview.md) and install core add-ons: